SkinAi_Flowchart.js

```diff
@@ -1,66 +1,171 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, useState } from "react";
```

Comment:
User React files inside ./client not in ./ root folder
react needs dependencies which is available at client

Reply:
ok